src/DynamoCoreWpf/UI/GuidedTour/Survey.cs

```diff
@@ -1,24 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows;
-using System.Windows.Controls.Primitives;
+﻿using Dynamo.Wpf.ViewModels.GuidedTour;
+using Dynamo.Wpf.Views.GuidedTour;
 
 namespace Dynamo.Wpf.UI.GuidedTour
 {
     public class Survey : Step
     {
+        public string LinkNexGuide { get; set; }
+
+        public string RatingTextTitle { get; set; }
+
+        public double ContentWidth { get; set; }
+
         public Survey(HostControlInfo host, double width, double height)
-            :base(host, width, height)
+            : base(host, width, height)
         {
             //In the Survey constructor we call the base constructor passing the host information and the popup width and height
         }
 
         protected override void CreatePopup()
         {
             //In this place will go the code for creating a SurveyWindow
```

Comment:
remove this comment?

Reply:
Removed in the next commit: https://github.com/DynamoDS/Dynamo/pull/11834/commits/84586419aa1ca8d940fcc13e6faff4ab4eba1968